docs/pages/extensibility/load-generators.md

```diff
@@ -9,11 +9,11 @@ language: en
 list: include
 ---
 
-Users may prefer to use one load generator over the next given the difference of capabilities between load generators, so Meshery provides a `load generator interface` (a gRPC interface) behind which a load generator can be implemented. Meshery provides users with choice of which load generator they prefer to use for a given performance test. Users may set their configure their own preference of load generator different that the default load generator.
+Users may prefer to use one load generator over the next given the difference of capabilities between load generators, so Meshery provides a `load generator interface` (a gRPC interface) behind which a load generator can be implemented. Meshery provides users with choice of which load generator they prefer to use for a given performance test. Users may configure their own preference for the load generator, different from the default load generator.
```

Comment:
Thanks for this, @ValGrace. Unfortunately, this rephrasing takes on a different meaning. We'll either need to leave the current phrasing or think of an alternative.

Reply:
Users do you have preference settings that they can make to modify how it is that their chosen low generator behaves. And this is something that can be documented. You can use playground.meshery.io to quickly see what preferences people can set. 

However, what is being discussed in this sentence is the fact that users can choose between load generators.